custom_components/wienerlinien/sensor.py

```diff
@@ -60,6 +61,7 @@ def __init__(self, api, name, firstnext):
         self._name = name
         self._state = None
         self.attributes = {}
+        self._unique_id = self._name
```

Comment:
a name is not unique

Reply:
for monitor + line it's good enough for me

Reply:
I don't think that a bus stop has the same line twice...

Reply:
In Norway (where I live) that's quite common actually, I don't know about this one.
I made this integration way back when for @Underknowledge

Reply:
> I don't think that a bus stop has the same line twice...
Hi, thank you very much this is going the right direction, but: the stations title is the same for bothe directions of the line - so the unique_id should be 
 a: station title + line name + line direction + firstnext 
 OR
b: station stop id + line name + firstnext

if we could get that different from the sensor name like so:
  line name + direction + "1.: " 
  line name + direction + "2.: "

would this brush up ecen more

thank you all very much for improving this - I'm on the icon thing  and when I got this right (still learning this syntax is killing me ^^) will gladly hand it over for implementation or not as you like